lib/flutter_html.dart

```diff
@@ -62,6 +62,7 @@ class Html extends StatefulWidget {
     this.onImageTap,
     this.tagsList = const [],
     this.style = const {},
+    this.selectable = false,
```

Comment:
We're moving away from `SelectableHtml` (see #1153). Please remove this `selectable` parameter from your changes so we can evaluate just the `placeholderAlignment` portion.